mbf_abstract_nav/src/controller_action.cpp

```diff
@@ -310,6 +310,13 @@ void ControllerAction::runImpl(GoalHandle &goal_handle, AbstractControllerExecut
         goal_handle.setAborted(result, result.message);
         break;
 
+      case AbstractControllerExecution::ROBOT_DISABLED:
+        ROS_FATAL_STREAM_NAMED(name_, "Robot stopped and ignored velocity commands!");
+        controller_active = false;
+        fillExePathResult(mbf_msgs::ExePathResult::ROBOT_STUCK,"Robot disabled and ignored velocity commands: ",result);
```

Comment:
```suggestion
        fillExePathResult(mbf_msgs::ExePathResult::ROBOT_STUCK,"Robot stopped and ignored velocity commands: ",result);
```
Maybe better not to say "stopped",,,  can be that it never moved on the first place!!!   What about?
"Robot ignoring velocity commands for more than N seconds"  ( u need to read the param here again;  but don't do if it looks ugly)

Reply:
Tiago said that it would be ugly if I want to read the param here. So probably it would be good to not do it here.